### PR TITLE
Add Edita's changes

### PR DESCRIPTION
**Pull request by KyleWong2510**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- No Edita

## What is the new behavior?

- Now Edita!  All UserInfo.js styling and testing completed

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: code/web/src/modules/common/header/Header.js [hunk 1]
```diff
@@ -1,90 +1,106 @@
...
-      boxShadow: level1,
-      padding: '0 2em',
-      height: '5em',
-      position: 'fixed',
-      left: 0,
-      right: 0,
-      top: 0
-    }}>
+    <header
+      style={{
```
Comment (sagemlee) [inline code/web/src/modules/common/header/Header.js:27]: Seems to be a lot of changes in this file, is this just spacing?

File: code/web/src/modules/user/api/actions.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ export function login(userCredentials, isLoading = true) {
     return axios.post(routeApi, query({
       operation: 'userLogin',
       variables: userCredentials,
-      fields: ['user {name, email, description, shippingAddress, shipDate, imageUrl, role, }', 'token']
+      fields: ['user {name, email, role, }', 'token']
```
Comment (sagemlee) [inline code/web/src/modules/user/api/actions.js:39]: http://gph.is/1aR7NWq

File: code/web/src/modules/common/header/Header.js (hunk 1 above)
Comment (edignot) [reply code/web/src/modules/common/header/Header.js:27]: It's spacing. Only real change I see os a logout button

File: code/web/src/modules/user/api/actions.js (hunk 2 above)
Comment (arpariseau) [reply code/web/src/modules/user/api/actions.js:39]: Gotta love how one small thing can break everything.

**Pull request merged**